hoot-core/src/main/cpp/hoot/core/io/OsmApiChangeset.cpp

```diff
@@ -261,21 +261,21 @@ void XmlChangeset::fixMalformedInput()
       long node_id = it->first;
       //  Set the node's status to failed if negative
       if (node_id < 1)
-        failNode(node_id);
+        failNode(node_id, true);
```

Comment:
Set the status to failed AND set the element as "sent" so that it doesn't try to resend and sent counter is up to date.